x-pack/legacy/plugins/ml/public/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -894,7 +896,7 @@ export class TimeSeriesExplorer extends React.Component {
 
   componentWillUnmount() {
     this.subscriptions.unsubscribe();
-    this.props.timefilter.off('timeUpdate', this.refresh);
+    this.timeUpdateSubscription.unsubscribe();
```

Comment:
If you use `subscriptions.add(...)` as suggested above you could just remove this line.

Reply:
done